webapp/packages/core-version-update/src/BaseUpdateInstruction.tsx

```diff
@@ -0,0 +1,27 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+import { Link, useTranslate } from '@cloudbeaver/core-blocks';
+
+import type { InstructionComponent } from './VersionUpdateService.js';
+
+const SPLITTER = '*splitter*';
```

Comment:
This splitter is confusing, why do we need it? Can we use \n or translate args?

Reply:
split to 2 locale keys. the best we can do here